resources.html

```diff
@@ -21,23 +21,30 @@ <h2 class="title">Editor</h2>
     </div>
 </div>
 <h2 class="title">Curriculum &amp; Lesson Plans</h2>
-<p>
-    In Progress - contact us for more info
-</p>
+<h3 class="title">AI/ML w/ North Hollywood Lesson Materials</h3>
+<ul>
+    <li><a href="https://drive.google.com/open?id=1yzc0K7pztHydPVXGULlYszm-J_OnqRxkjr5PlQ4RHDc">Lesson 1 Slides - What is AI?</a></li>
+    <li><a href="https://drive.google.com/open?id=1eDwEFXscmPcLhBuYd9ar0gSkjtiyFay55q4IpwgbA5w">Lesson 2 Slides - What is ML?</a></li>
+    <li><a href="https://drive.google.com/open?id=12M3ApbtPJPZsLugfH-OObV1tvkMzWJtO">Lesson 3 Jupyter Notebook - introduction to plotting data</a></li>
```

Comment:
Could we do "Lesson #: ..." instead of "Lesson # - ..."? And capitalize "introduction to plotting data"?